presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -651,6 +651,24 @@ void PrestoServer::initializeThreadPools() {
   }
 }
 
+std::unique_ptr<cache::SsdCache> PrestoServer::setupSsdCache() {
```

Comment:
```
VELOX_CHECK_NULL(cacheExecutor_);
```

Reply:
Is this needed right after we created cacheExecutor_? Also we are already checking it inside SsdCache.